generators/server/templates/src/main/java/package/config/SecurityConfiguration_reactive.java.ejs

```diff
@@ -277,6 +277,13 @@ public class SecurityConfiguration {
             .pathMatchers("/api/auth-info").permitAll()
             .pathMatchers("/api/admin/**").hasAuthority(AuthoritiesConstants.ADMIN)
             .pathMatchers("/api/**").authenticated()
+<%_ if (applicationTypeGateway) { _%>
+  <%_ if (microfrontend && authenticationTypeJwt) { _%>
```

Comment:
What's different about JWT vs OAuth?

Reply:
Not sure, js are been denied even logged in.
Since OAuth2 is passing at CI, I think this is a JWT problem only.
I have to track this behavior better.

Reply:
I am only using JWT, because last time I checked keycloak was a no go at Mac M1 docker.

Reply:
That's indeed required. Due to missing bearer-token in the request.

Reply:
Our client framework passes to gateway.
https://github.com/jhipster/generator-jhipster/blob/5de7cdebdc7e2cf2338779b076f52d07cdb97caf/generators/client/templates/angular/src/main/webapp/app/core/interceptor/auth.interceptor.ts.ejs#L44

Since this is done by webpack, no bearer token is passed.